ui/app/clinical/constants.js

```diff
@@ -18,6 +18,10 @@ Bahmni.Clinical.Constants = (function () {
         age: "Age",
         weight: "Weight"
     };
+    var careSetting = {
```

Comment:
The changes added to this file here is not relevant to this PR. It would be appreciated if this could be a separate PR to allow us to track the changes more effectively, enabling both review and testing of this new feature.